src/main/java/TaskList.java

```diff
@@ -0,0 +1,82 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskList {
+    private static final String indent = "    ";
```

Comment:
Should this constant follow another capitalization convention? Perhaps you can consider simply using the same `indent` constant declared in the `Duke` class?
```suggestion
    private static final String INDENT = "    ";
```